components/Comment/lib/types.ts

```diff
@@ -0,0 +1,53 @@
+import { parseCreatedBy } from "~/config/types/contribution";
+import { CreatedBy, ID } from "~/config/types/root_types";
+import { formatDateStandard, timeSince } from "~/config/utils/dates";
+
+export enum POST_TYPES {
+  DISCUSSION = "DISCUSSION",
+  SUMMARY = "SUMMARY",
+  REVIEW = "REVIEW",
+  ANSWER = "ANSWER",
+}
+
+export type Comment = {
+  id: ID;
+  createdDate: string;
+  updatedDate: string;
+  timeAgo: string;
+  createdBy: CreatedBy | null;
+  content: any;
```

Comment:
`string | undefined` might be a safer bet here, looking at how this field is used

Reply:
You know what, this should have been `object` type. It was my bad defaulting it to a string. I updated it in my follow-up PR branch https://github.com/ResearchHub/researchhub-web/blob/ad700c21f819a126c39cb02a902c73c6e6b10a58/components/Comment/CommentEditor.tsx#L15-L22